spec/shale/mapper_spec.rb

```diff
@@ -8,14 +8,14 @@ module ShaleMapperTesting
   BAR_DEFAULT_PROC = -> { 'bar' }
 
   class Parent < Shale::Mapper
-    attribute :foo, Shale::Type::String
```

Comment:
Can you also add a test case using classes?

Reply:
For this case, I changed this attribute back to the class so that we had a mix of the two in the tests rather than copying a large chunk of the test suite and using the classes. Let me know if that seems sufficient or if you had something else in mind.

Reply:
that's what I had in mind, thank you